client/ayon_core/hosts/maya/plugins/publish/extract_pointcache.py

```diff
@@ -152,24 +202,147 @@ def process(self, instance):
+        override_defs = {
+            "attr": {
+                "def": TextDef,
+                "kwargs": {
+                    "label": "Custom Attributes",
+                    "placeholder": "attr1; attr2; ...",
+                }
+            },
+            "attrPrefix": {
+                "def": TextDef,
+                "kwargs": {
+                    "label": "Custom Attributes Prefix",
+                    "placeholder": "prefix1; prefix2; ...",
+                }
+            },
+            "dataFormat": {
+                "def": EnumDef,
+                "kwargs": {
+                    "label": "Data Format",
+                    "items": ["ogawa", "HDF"],
+                }
+            },
+            "melPerFrameCallback": {
+                "def": TextDef,
+                "kwargs": {
+                    "label": "melPerFrameCallback",
+                }
+            },
+            "melPostJobCallback": {
+                "def": TextDef,
+                "kwargs": {
+                    "label": "melPostJobCallback",
+                }
+            },
+            "preRollStartFrame": {
+                "def": NumberDef,
+                "kwargs": {
+                    "label": "Start frame for preroll",
+                    "tooltip": (
+                        "The frame to start scene evaluation at. This is used"
+                        " to set the starting frame for time dependent "
+                        "translations and can be used to evaluate run-up that"
+                        " isn't actually translated."
+                    ),
+                }
+            },
+            "pythonPerFrameCallback": {
+                "def": TextDef,
+                "kwargs": {
+                    "label": "pythonPerFrameCallback",
+                }
+            },
+            "pythonPostJobCallback": {
+                "def": TextDef,
+                "kwargs": {
+                    "label": "pythonPostJobCallback",
+                }
+            },
+            "userAttr": {
+                "def": TextDef,
+                "kwargs": {
+                    "label": "userAttr",
+                }
+            },
+            "userAttrPrefix": {
+                "def": TextDef,
+                "kwargs": {
+                    "label": "userAttrPrefix",
+                }
+            },
+            "visibleOnly": {
+                "def": BoolDef,
+                "kwargs": {
+                    "label": "Visible Only",
+                }
+            }
+        }
+
+        defs = super(ExtractAlembic, cls).get_attribute_defs()
+
+        defs.extend([
+            UISeparatorDef("sep_alembic_options"),
+            UILabelDef("Alembic Options"),
+        ])
+
+        # The Arguments that can be modified by the Publisher
+        overrides = set(getattr(cls, "overrides", set()))
+
+        # What we have set in the Settings as defaults.
+        flags = set(getattr(cls, "flags", set()))
```

Comment:
Since these are exposed as regular attributes on the class by its definition, why use `getattr`?

```suggestion
        # The Arguments that can be modified by the Publisher
        overrides = set(cls.overrides)

        # What we have set in the Settings as defaults.
        flags = set(cls.flags)
```